### ISSUE TITLE
PDF / excel extraction of the final results for simple use

### ISSUE THREAD

**Issue by ARPAV2022** [isa-group/wecaremed#18]
According to us an additional feature could be add to allow the final user to download in pdf /excel (or both format) the results of the calculation with all the data entry. This could be useful in order to have a "piece of paper" to provide exhaustive information behind the CF number of the project. 
Even if it is not clear what kind of demonstration and reporting effort will be required by the JS to the future applicants, this additional feature could already provide a solution.
In other words, we suggest to integrate a button at the bottom of the page in order to stamp /download the data entry in pdf or excel format.
What do you think? Is it feasible?

**Comment by MaToSan24** [comment]
A "Generate PDF" button has been implemented, feel free to check it out.
![image](https://user-images.githubusercontent.com/63660411/157039283-0f48e7fe-c967-4244-916b-98a59dff093b.png)
